src/topojson/topojson.ml

```diff
@@ -242,8 +235,33 @@ module Make (J : Intf.Json) = struct
              ("arcs", J.array (J.array (J.array J.int)) arcs);
            ]
           @ properties_or_null properties
-          @ bbox_to_json_or_empty bbox
-          @ foreign_members)
+          @ bbox_to_json_or_empty bbox @ foreign_members)
+    end
+
+    module Accessor = struct
```

Comment:
I think it would be more convenient if we didn't nest these inside an `Accessor` module. I think I did that for GeoJSON but probably shouldn't have.